### DISCUSSION TITLE
NullReferenceExceptions

### DISCUSSION THREAD

**Discussion by bryfa** [oqtane/oqtane.framework#803] [General]
Hi all,

I just started to look into Oqtane. I really like the framework and the chosen architecture. While trying things out I get NullReference exceptions reqularly.  So I switched the  projects to C#8 and enabled nullable checks. I used this on other projects and it helped me a lot. Just enable it and fix all you warnings.. It will not eliminate all your reference exceptions though. It will improve you code base for sure.

![image](https://user-images.githubusercontent.com/8579202/96212370-fec7da00-0f76-11eb-8a8e-42d0444fbfd3.png)

**Comment by hishamco** [comment]
I will have a look

**Reply by TonyValenti** [reply]
I can definitely recommend enabling nullable.  We've started doing that and the number of bugs it catches is amazing.
